src/visualization/write2vtk.jl

```diff
@@ -181,11 +181,12 @@ function write2vtk!(vtk, v, u, t, system; write_meta_data=true)
 end
 
 function write2vtk!(vtk, v, u, t, system::FluidSystem; write_meta_data=true)
-    vtk["velocity"] = view(v, 1:ndims(system), :)
+    vtk["velocity"] = [current_velocity(v, system, particle)
+                       for particle in each_moving_particle(system)]
     vtk["density"] = [particle_density(v, system, particle)
-                      for particle in eachparticle(system)]
+                      for particle in each_moving_particle(system)]
     vtk["pressure"] = [particle_pressure(v, system, particle)
-                       for particle in eachparticle(system)]
+                       for particle in each_moving_particle(system)]
```

Comment:
The coordinates are those of `active_particles`, so this here should use the exact same iterator.
For now, we don't have any active fixed particles for the fluid, so it's equivalent, but this might change.

Same below.